parser/parser.go

```diff
@@ -755,6 +761,114 @@ func (p *Parser) parsePoryswitchMovementCases() (map[string][]string, error) {
 	return movementCases, nil
 }
 
+func (p *Parser) parseMartStatement() (*ast.MartStatement, error) {
+	statement := &ast.MartStatement{
+		Token:     p.curToken,
+		MartItems: []string{},
+	}
+	scope, err := p.parseScopeModifier(token.LOCAL)
+	if err != nil {
+		return nil, err
+	}
+	statement.Scope = scope
+	if err := p.expectPeek(token.IDENT); err != nil {
+		return nil, fmt.Errorf("line %d: missing name for mart statement", p.curToken.LineNumber)
+	}
+
+	statement.Name = &ast.Identifier{
+		Token: p.curToken,
+		Value: p.curToken.Literal,
+	}
+
+	if err := p.expectPeek(token.LBRACE); err != nil {
+		return nil, fmt.Errorf("line %d: missing opening curly brace for mart '%s'", p.peekToken.LineNumber, statement.Name.Value)
+	}
+	p.nextToken()
+	statement.MartItems, err = p.parseMartValue(true)
+	if err != nil {
+		return nil, err
+	}
+
+	return statement, nil
+}
+
+func (p *Parser) parseMartValue(allowMultiple bool) ([]string, error) {
+	martCommands := make([]string, 0)
+	for p.curToken.Type != token.RBRACE {
+		if p.curToken.Type == token.PORYSWITCH {
+			poryswitchCommands, err := p.parsePoryswitchMartStatement()
+			if err != nil {
+				return nil, err
+			}
+			martCommands = append(martCommands, poryswitchCommands...)
+		} else if p.curToken.Type == token.IDENT {
+			martCommand := p.curToken.Literal
```

Comment:
It might be nice if the `mart` operator supported constants.  Not sure how useful it would be.  I'm referring to Poryscript constants defined in the scripts with `const`.

Reply:
Happy to add it, but does `movement` support constants as well?

Reply:
No, `movement` doesn't and probably shouldn't support constants.